test/test_accounting_cli.py

```diff
@@ -111,7 +111,7 @@ def setUpClass(self):
 
     # add a valid user to association_table
     def test_01_add_valid_user(self):
-        aclif.add_user(conn, "fluxuser", "1", "acct", "10", "100", "60")
+        aclif.add_user(acct_conn, "fluxuser", "1", "acct", "10", "100", "60")
```

Comment:
Hmmm. I am confused as to how this change was possible... What commit introduced the connection name change from `conn` to `acct_conn`? Doesn't this change belong that commit as opposed to this commit?

Reply:
I believe this change was made in db8d0282f516ecae4424625374c550f297ec6b2f, right? When all of the unit tests for the subcommands were added? Is there another commit that also makes this change?

Reply:
Yes. But I am curious about its previous commit. Was `conn` a valid variable in the previous commit? The reason I ask is, at the current commit db8d0282f516ecae4424625374c550f297ec6b2f, I don't see your change to define `acct_conn`. You just used it; that means that variable was probably defined previously. 

If that's the case, should the previous commit also not have the reference of `conn` but `acct_conn` instead? As is, I'm a bit concerned that the previous commit may have unintentionally broken.

Reply:
The first commit that introduced the `acct_conn` variable was 374242d4f6db01468d65bf376d04d6e888c7a146 - this is from the PR that added the front end to the job archive (#33), as two separate connections were needed (one to the jobs DB, and one to the accounting DB).

Reply:
Hmmm. From your 374242d, I see you made the necessary change:

```python
 aclif.add_user(acct_conn, "fluxuser", "1", "acct", "pacct", "10", "100", "60")
```

So why the diff says you made `conn` to `acct_conn` in the current commit?

Reply:
Ah I actually see the changes from 3c1951f75e46176d2f7b323fa2899960af1136cd. Perhaps the changes in that commit is incorrect?

Reply:
I'm not sure. I know I had to rebase this PR after #33, but I'm not sure why the change is recognized in this most recent commit...

Reply:
Ah; so I guess I made the change to `acct_conn` in 374242d, reverted it in 3c1951f, and then brought it back in the last commit, db8d028?

Reply:
My guess is this went wrong through rebasing and conflict resolution. Please drop that test script change in 3c1951f (you can use git rebase -i -- then edit) and see if that can clear up this churn.

Reply:
You are probably right - so sorry for the confusion. Just pushed an update.